spring-xd-rest-client/src/main/java/org/springframework/xd/rest/client/ResourceOperations.java

```diff
@@ -50,4 +50,10 @@
 	 */
 	public void destroyAll();
 
+	/**
+	 * Delete Queues and DLQs for a stream, and any exchanges created for taps.
```

Comment:
We should make is bus-agnostic - will fix during merge; something like:

"Remove any resources that cannot automatically be cleaned up during undeploy"

We might need to extend it to other bus implementations in future.